### PR TITLE
Temporarily show the latest entered character in an obscured text field

### LINKED ISSUE #11817: Show last typed character briefly in TextField when obscureText is true
This request came in from Posse.

iOS and Android password fields both support this I believe?

### PR DESCRIPTION
**Pull request by jason-simmons**
Fixes https://github.com/flutter/flutter/issues/11817

### CODE REVIEW

**Comment by jason-simmons** [comment]
@Hixie

File: packages/flutter/lib/src/widgets/editable_text.dart [hunk 1]
```diff
@@ -516,8 +525,15 @@ class EditableTextState extends State<EditableText> with AutomaticKeepAliveClien
...
   Duration get cursorBlinkInterval => _kCursorBlinkHalfPeriod;
 
+  int _obscureShowCharTicksPending = 0;
+  int _obscureLatestCharIndex;
+
   void _cursorTick(Timer timer) {
     _showCursor.value = !_showCursor.value;
+    if (_obscureShowCharTicksPending > 0) {
+      _obscureShowCharTicksPending--;
+      setState(() {});
```
Comment (Hixie) [inline packages/flutter/lib/src/widgets/editable_text.dart]: Please put a comment inside the `{}` explaining what state changed (or, better, put the actual state change there).

**Comment by Hixie** [comment]
needs a test

**Comment by jason-simmons** [comment]
done - PTAL

File: packages/flutter/lib/src/widgets/editable_text.dart [hunk 2]
```diff
@@ -516,8 +525,14 @@ class EditableTextState extends State<EditableText> with AutomaticKeepAliveClien
   @visibleForTesting
   Duration get cursorBlinkInterval => _kCursorBlinkHalfPeriod;
 
+  int _obscureShowCharTicksPending = 0;
+  int _obscureLatestCharIndex;
+
   void _cursorTick(Timer timer) {
     _showCursor.value = !_showCursor.value;
+    if (_obscureShowCharTicksPending > 0) {
+      setState(() { _obscureShowCharTicksPending--; });
```
Comment (Hixie) [inline packages/flutter/lib/src/widgets/editable_text.dart]: minor issue, feel free to ignore: consider `-= 1` rather than postfix `--`

**Comment by Hixie** [comment]
LGTM

**Pull request merged**